### PR TITLE
Add PropagateAtLaunch to Tag

### PR DESCRIPTION
**Pull request by w0rmr1d3r**
### Additions
- Create `TagProperty`, child class of `Tag`.
### Updates
- Update `CLOUDFORMATION_ACTIONS`.

### CODE REVIEW

**Review by ocrawford555 — DISMISSED** [review]

**Review by ignaciobolonio — DISMISSED** [review]

**Review by ignaciobolonio — DISMISSED** [review]

File: pycfmodel/model/resources/properties/tag.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
+from typing import Optional
+
 from pycfmodel.model.resources.properties.property import Property
-from pycfmodel.model.types import ResolvableStr
+from pycfmodel.model.types import ResolvableBool, ResolvableStr
 
 
 class Tag(Property):
```
Comment (oscarbc96) [inline pycfmodel/model/resources/properties/tag.py:7]: Shall we create a child class adding this new property? This property is just for `AWS::AutoScaling::LaunchConfiguration` and doesn't apply to all other places.
(Is this detected in another object?)

Comment (w0rmr1d3r) [reply pycfmodel/model/resources/properties/tag.py]: Like the idea! See [4b81f7f](https://github.com/Skyscanner/pycfmodel/pull/92/commits/4b81f7f8f215cd969db636ed85f54084eed2615e) for the changes.
With this change, we will need to update some internal rules as well.

**Comment by oscarbc96** [comment]
Probably include it also at: https://github.com/Skyscanner/pycfmodel/blob/master/pycfmodel/model/resources/properties/types.py#L12

However, not sure if we should create a specific resource of `AWS::AutoScaling::LaunchConfiguration`, so it always uses the property tag as generic may create tags with Tag and Property tag

**Comment by w0rmr1d3r** [comment]
Closing the PR since it doesn't bring much value and we don't need this new resource/property at the moment.
Can be reopen if needed.

**Pull request closed without merging**